migration/v0.19.x-v0.20.x/upgrade-thanos-receiver-pvc.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+sc_config="${CK8S_CONFIG_PATH}/sc-config.yaml"
+
+upgrade_receiver_pvc() {
+    value=$(yq r "$1" "$2" | grep -o '[[:digit:]]*')
+
+    if [[ "${value}" -ge 50 ]]; then
+        echo "The PVC size for thanos receiver is set to: ${value}. This size is greater or equal with the new default, 50Gi. Skipping."
+    else
+        echo "The PVC size for thanos receiver than the new default, 50Gi. It will be updated."
+        ./bin/ck8s ops kubectl sc delete sts --cascade=orphan -n thanos thanos-receiver-receive
+        ./bin/ck8s ops kubectl sc patch pvc data-thanos-receiver-receive-0 -n thanos --type='json' -p="\"[{'op': 'replace', 'path': '/spec/resources/requests/storage', 'value':'50Gi'}]\""
+        ./bin/ck8s ops kubectl sc patch pvc data-thanos-receiver-receive-1 -n thanos --type='json' -p="\"[{'op': 'replace', 'path': '/spec/resources/requests/storage', 'value':'50Gi'}]\""
```

Comment:
Doesn't we need to apply the thanos receiver chart again? otherwise the sts wouldn't be recreated

Reply:
It is covered by step 14 in the migration guide.

Reply:
Right, then I'm fine